server/api/jurisdictions.py

```diff
@@ -55,14 +55,38 @@ def round_status_by_jurisdiction(
         .values(AuditBoard.jurisdiction_id, func.count())
     )
 
-    sampled_ballot_count_by_jurisdiction = dict(
+    ballots_in_round = (
+        SampledBallot.query.join(SampledBallotDraw)
+        .filter_by(round_id=round.id)
+        .distinct(SampledBallot.id)
+        .subquery()
```

Comment:
you could probably do this with a `count(distinct sampled_ballot_id)` and just query `SampledBallotDraw`. Not critical, perf might be identical.